internal/integration_test/spectest/v2/spec_test.go

```diff
@@ -0,0 +1,31 @@
+package spectest
+
+import (
+	"embed"
+	"testing"
+
+	"github.com/tetratelabs/wazero/internal/integration_test/spectest"
+	"github.com/tetratelabs/wazero/internal/wasm"
+	"github.com/tetratelabs/wazero/internal/wasm/interpreter"
+	"github.com/tetratelabs/wazero/internal/wasm/jit"
+)
+
+//go:embed testdata/*.wasm
+//go:embed testdata/*.json
+var testcases embed.FS //nolint:unused
+
+const enabledFeatures = wasm.Features20220419
+
+func TestJIT(t *testing.T) {
+	t.Skip() // TODO!
```

Comment:
skipped the actual test for now as I found that I need to patch several places in a different PRs.